projects/showcase/test/specs/alphanumeric-layout.spec.ts

```diff
@@ -69,15 +69,7 @@ describe("AlphanumericLayout", () => {
     it("Click on 'Caps Lock' virtual keyboard key to change layout to uppercase", async () => {
         await VirtualKeyboard.get().clickCapsLock();
 
-        await ReportUtility.addExpectedResult("Virtual keyboard now shows all letters in uppercase and more symbols", async() => {
-            expect(await VirtualKeyboard.get().isPresent()).toBeTruthy();
-            expect(await VirtualKeyboard.get().getRowCount()).toEqual(5);
-            expect(await VirtualKeyboard.get().getRowKeys(0)).toEqual([      '~', '!', '@', '#', '$', '%', '^', '&amp;', '*', '(', ')', '_', '+', '{bksp}']);
-            expect(await VirtualKeyboard.get().getRowKeys(1)).toEqual([  '{tab}', 'Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P', '{', '}', '|']);
-            expect(await VirtualKeyboard.get().getRowKeys(2)).toEqual([ '{lock}', 'A', 'S', 'D', 'F', 'G', 'H', 'J', 'K', 'L', ':', '"', '{enter}']);
-            expect(await VirtualKeyboard.get().getRowKeys(3)).toEqual(['{shift}', 'Z', 'X', 'C', 'V', 'B', 'N', 'M', '&lt;', '&gt;', '?', '{shift}']);
-            expect(await VirtualKeyboard.get().getRowKeys(4)).toEqual(['{space}']);
-        });
+        await expectAlphanumericUppercaseLayout();
```

Comment:
I am OK on moving these expectations into a shared method to faciliate reusing it. 
However, I would suggest to create a dedicated class for that.
I would suggest to place them into a `expectation/layout.expectation.ts` file in a class named `LayoutExpectation`. Then, the methods should be converted to `static`, so we don't need to create a class for them.

Reply:
Also for consistency, the numeric layout expectation should be placed there too.